kotest-assertions/src/commonMain/kotlin/io/kotest/tuples.kt

```diff
@@ -0,0 +1,38 @@
+package io.kotest
```

Comment:
Should this really be at the root package?

Reply:
I guess not. Moved it.